.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Learn more about EditorConfig at http://editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+trim_trailing_whitespace = true
+end_of_line = lf
+insert_final_newline = true
+
+[*.{clj,cljs,cljc}]
+indent_style = space
+indent_size = 2
```

Comment:
There is no newline here; which should break CI because this file is non-compliant